### ISSUE TITLE
My chrome can't use lantern, but IE can.

### ISSUE THREAD

**Issue by yz-nju** [getlantern/lantern#3777]
Help!
My chrome version is 49.0.2623.87 m.
With chrome, I can't access blocked  sites, but I can with IE. I've tried SwitchOmega, but doesn't work. So, what can I do?

**Comment by myleshorton** [comment]
Does it work if you disable all of your extensions @yz-nju? You can also check if Chrome has marked Lantern as a "bad proxy" in chrome://net-internals/#proxy

**Comment by shingoxray** [comment]
I have the same issue. Does not work with Chrome or Edge. But IE is OK.

**Comment by myleshorton** [comment]
Do you have any extensions running with Chrome @shingoxray?

**Comment by shingoxray** [comment]
@myleshorton Yes, but it works some time before, with extensions. And Edge has no extension.

**Comment by Chao1155** [comment]
I met the same problem before.
You may try to turn on options like "system proxy" in your extension, which
redirect accesses to that lantern port used in your localhost.

On Fri, Mar 25, 2016 at 10:50 AM shingoxray <notifications@github.com>
wrote:

> @myleshorton <https://github.com/myleshorton> Yes, but it works some time
> before, with extensions. And Edge has no extension.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/getlantern/lantern/issues/3777#issuecomment-201114931>
>
-- 
Regards,
Chao

**Comment by shingoxray** [comment]
I've tried this, system proxy, still not work. Weird that it worked before, nothing changed. @Chao1155

**Comment by yz-nju** [comment]
I've already uninstall all extensions, and still don't work. And when I check chrome://net-internals/#proxy, it's ok，example:
![11111111](https://cloud.githubusercontent.com/assets/17826968/14042674/2a493a8e-f2b7-11e5-84b9-fc53d12ae050.png)

I've found when I use lantern on chrome, it tells me "waiting for proxy tunnel".
And when I use Edge, I even can't enter http://ui.lantern.io/, example:
![qq 20160325182331](https://cloud.githubusercontent.com/assets/17826968/14042640/c74bd54a-f2b6-11e5-8d06-2bdf4513b45d.png)
@myleshorton

**Comment by JiejiaH** [comment]
I have the same question with you until I uninstall the xunlei.exe.

**Comment by shingoxray** [comment]
Yes! It's Thunder/Xunlei. Uninstall it then the problem is gone.

**Comment by yz-nju** [comment]
yeah, it worked!
大兄弟，谢谢昂@JiejiaH

**Comment by myleshorton** [comment]
Thanks @JiejiaH @shingoxray and @yz-nju! @fffw have you been able to discover when about Thunder/Xunlei causes problems?

**Comment by fffw** [comment]
Unfortunately I didn't have a chance to look at it, but will do once finished current UI work @myleshorton. My impression is that only CONNECT request is affected by Xunlei but not 100% sure.

**Comment by shingoxray** [comment]
@myleshorton Once you install Thunder/Xunlei, it will always causes the problem, until it is uninstalled.

**Comment by aranhoide** [comment]
@fffw assigning to you since you said you'll look at this.

Setting to closest milestone that is not due yet.  I'll call this a P2 because incompatibilities will happen, and while fixing them or working around them would be great, just being aware of them so we can help users in their troubleshooting may be the best we can do in many cases.

**Comment by xiam** [comment]
Probably related to https://github.com/getlantern/lantern/issues/3862
